src/components/DatePicker/RelativeDateRange.jsx

```diff
@@ -220,13 +233,22 @@ export const RelativeDateRange = ({
     onChange,
     onSubmit,
     timezoneName,
+    isFutureDatesAllowed = false, // New prop
```

Comment:
// New prpo => maynot needed. 
you can add comment of what it does, or add in Ui lit story board https://ui.xola.io/?path=/docs/data-display-date-time-date-range-picker--relative-date-ranges